test/functional/view-models.js

```diff
@@ -60,6 +60,41 @@ describe('View all models', function () {
         });
     });
 
+    it('adds a new item to the model', () => {
+        mockServer.expect({
+            method: 'post',
+            endpoint: 'item',
+            json: {
+                modelAddress: models[0].address
+            },
+            response: {
+                status: 'success',
+                data: {
+                    address: 'iGwEZVvgu',
+                    modelName: 'Resistor'
+                }
+            }
+        });
+        mockServer.expect({
+            method: 'get',
+            endpoint: 'model/all',
+            response: {
+                status: 'success',
+                data: {
+                    models
+                }
+            }
+        });
+        return app.client.click('.btnAddItemToModel:nth-of-type(1)').then(() => {
+            return app.client.waitForVisible('.modal', 5000);
+        }).then(() => {
+            return app.client.click('.modal .modal-content button[type="button"]');
+        }).then(()=> {
+            return app.client.waitForVisible('.toast', 1000);
+        }).then(() => {
+            mockServer.validate();
```

Comment:
I can't tell which model is clicked because the list could be different every time. It's also not the client's job to add to the quantity, so it makes no sense to assert that here.